test/Microsoft.TestPlatform.CommunicationUtilities.PlatformTests/SocketClientTests.cs

```diff
@@ -38,9 +38,11 @@ public void Dispose()
 #if NETFRAMEWORK
             // tcpClient.Close() calls tcpClient.Dispose().
             this.tcpClient?.Close();
+            GC.SuppressFinalize(this);
```

Comment:
```suggestion
```

Reply:
resolved added GC.SuppressFinalize(this); after the #endif